glowbot/main.py

```diff
@@ -60,4 +60,5 @@ def run_discord_bot():
     )
 
 if __name__ == '__main__':
+    run_discord_bot()
```

Comment:
I think this is left in here by accident?  `poetry` calls this via `poetry run glowbot-discord` so I don't think this is necessary.

Reply:
Yes it was. Sigh.